migrations/README.md

```diff
@@ -1,7 +1,12 @@
 # Database migration scripts
 
-Add new migrations with the filename format of the current version
-followed by a description of the migration being performed.
+Add new migrations with the filename format of the date at which the
+migration was created (without dashes), followed by the new version of
+the schema, followed by a description of the migration being performed.
+For example:
+
+`20190726-0.0.2-new-migration-description.js`
 
 Note that migration scripts are not transpiled with Babel. This could be
-addressed by introducing a custom resolver in `db-migrate.js`.
+addressed by adding Babel to the runtime dependencies if there is a
```

Comment:
Note that we're not pulling Babel over from devDependencies at this time to reduce the size of the bundle and to help keep us from being tempted to add runtime transpilation unless we have a good reason.

For now, migrations aren't a good enough reason; they might be if they would be too complex otherwise (though if DB migration scripts are getting that complex, we need to ask ourselves why.)